producer/producer_protocol_handler.go

```diff
@@ -251,7 +251,11 @@ func (w *BaseProducerProtocolHandler) HandleProposal(ph abstractprotocol.Protoco
 			err_log_event = fmt.Sprintf("Error creating message target: %v", err)
 		} else {
 			handled = true
-			if r, err := ph.DecideOnProposal(proposal, w.ec.GetExchangeId(), exchange.GetOrg(w.ec.GetExchangeId()), runningBCs, messageTarget, w.sendMessage); err != nil {
+			producerPol, err := persistence.FindNodePolicy(w.db)
+			if err != nil {
+				glog.Errorf("Error getting node policy from db: %v", producerPol)
```

Comment:
Please follow the pattern of other glog.Errorf calls in this file

Reply:
Fixed